apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/ClaimTypeV2.java

```diff
@@ -111,7 +111,7 @@ public enum ClaimTypeV2 {
    * @param serviceEndAttributeFunction the service end attribute function
    * @param entityLazyAttributes the value to use for {@link #getEntityLazyAttributes()}
    */
-  private ClaimTypeV2(
+  ClaimTypeV2(
```

Comment:
Enum constructors are effectively private.  No point in remove it form the declaration here.

https://docs.oracle.com/javase/specs/jls/se8/html/jls-8.html#jls-8.9.2

"In an enum declaration, a constructor declaration with no access modifiers is private."

Reply:
interestingly enough, I hadn't removed it; so as an experiment I added it back in and saved...voila! The formatter is doing this sort of stuff.